Documentation/UpdatingToGA.md

```diff
@@ -9,8 +9,36 @@ Between the RC2 and GA releases of the Microsoft Mixed Reality Toolkit, changes
 
 Since the release of RC2, there have been a number of API changes including some that may break existing projects. The following sections describe the changes that have occurred between the RC2 and GA releases.
 
+- [Event System](#event-system)
 - [Spatial Awareness](#spatial-awareness)
 
+### Event System
+
+**Changes**
+
+- The `IMixedRealityEventSystem` old API methods `Register` and `Unregister` has been marked as Obsolete. They are preserved for backward compatibility.
+- `InputSystemGlobalListener` has been marked as obsolete. Its functionality has not changed.
+- `BaseInputHandler` base class has been changed from `InputSystemGlobalListener` to `InputSystemGlobalHandlerListener`. This is a breaking change for any descendants of `BaseInputHandler`.
+
+**Motivation behind the change**
+
+Old event system Api `Register` and `Unregister` could potentially cause multiple issues in runtime, main being:
+
+- If a component registers for global events, it would receive global input events of ALL types.
```

Comment:
```suggestion
- If a component registers for global events, it would receive global input events of *all* types.
```

use italic for highlighting , not uppercase